<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConsultaAdreca</class>
 <widget class="QDialog" name="DialogConsultaAdreca">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>104</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consulta. Selecció d'adreça</string>
  </property>
  <widget class="QLabel" name="lVial">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>11</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vial</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leVial">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>11</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lNumero">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>39</y>
     <width>37</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero</string>
   </property>
  </widget>
  <widget class="QLabel" name="lLletra">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>27</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lletra</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leLletra">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNumero">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogConsultaAdreca</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogConsultaAdreca</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
